packages/data-edge/.gitattributes

```diff
@@ -0,0 +1 @@
+*.sol linguist-language=Solidity
```

Comment:
We can safely remove this file, it's from a time where GitHub had trouble with Solidity